test/test_conversion.py

```diff
@@ -62,6 +62,10 @@ def test_3to2_conversion(self):
         assert doc2.componentDefinitions[0].sequences[0] == 'https://synbiohub.org/public/igem/BBa_J23101_sequence'
         assert doc2.sequences[0].encoding == 'http://www.chem.qmul.ac.uk/iubmb/misc/naseq.html'
         assert doc2.sequences[0].elements == 'tttacagctagctcagtcctaggtattatgctagc'
+        for c in doc2.componentDefinitions:
+            for sa in c.sequenceAnnotations:
+                for loc in sa.locations:
+                    assert loc.orientation != 'http://sbols.org/v3#inline'
```

Comment:
Can we also have a positive check that we _do_ have some SBOL2 inline marks where expected? Orientation is optional, and so we need to make sure we're actually exercising the test - I don't know if the test file has an orientation in it at all.